webapp/src/main/java/io/github/microcks/repository/InvocationLogRepository.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright The Microcks Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.github.microcks.repository;
+
+import io.github.microcks.domain.InvocationLogEntry;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.repository.NoRepositoryBean;
+import java.util.List;
+
+/**
+ * Repository for InvocationLogEntries
+ */
+@NoRepositoryBean
```

Comment:
This construct and way to define repository is different than the one we used for `DailyStatisticRepository`. I find it a bit more difficult to understand - especially the `InvocationLogRepositoryImpl` part with the `MongoEntityInformation` override.

I think it could be nice to refactor reusing the same pattern here. What do you think?